joomla-projects/privacy-framework
# Change the backend icon - com_privacy

### Summary of Changes
Just a cosmetic change.
Changed the "icon" of com_privacy administration from a generic "dashboard" icon to a more meaningful "lock" to keep the "privacy" theme.


### Testing Instructions
Nothing special. Navigate in the backend of the com_privacy and see the lock icon instead of the dashboard one.


### Expected result
![immagine](https://user-images.githubusercontent.com/733133/42684061-50ff0ef6-868f-11e8-8357-0c84544ca767.png)


### Actual result
![immagine](https://user-images.githubusercontent.com/733133/42684032-435f7420-868f-11e8-8b24-c74097d941dd.png)


### Documentation Changes Required
None.
